client/components/keyed-suggestions/style.scss

```diff
@@ -6,6 +6,8 @@
 	margin-left: -1px;
 	overflow-y: auto;
 	max-height: 50vh;
+    border: 1px solid var( --color-neutral-10 );
+    border-top: 0;
```

Comment:
Small nitpick: The rest of this file is indented with tabs, but these new lines (and other new lines in this PR) are indented with spaces.
![2021-07-27_09-17](https://user-images.githubusercontent.com/937354/127170192-f46f7937-06b7-4375-a1d0-681c2a66549f.png)

Reply:
ups, my bad! fixed